input/fsh/SD_Medications.fsh

```diff
@@ -2,46 +2,24 @@ Profile:  CancerRelatedMedicationRequest
 Parent:   USCoreMedicationRequest
 Id:       mcode-cancer-related-medication-request
 Title:    "Cancer-Related Medication Request"
-Description:    "A record of the medication prescription or order for a patient whose condition is related to a primary or secondary cancer condition. The request may be reported by the prescriber, prescribing organization, or patient does not have to be directly observed.
+Description:    "A record of a medication prescription, administration, or consumption used to treat cancer. The medication may reported by the prescriber, prescribing organization, or patient, and does not have to be directly observed.
 
 Conformance statement:
 
- MedicationRequest resources associated with an mCODE patient SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
+ MedicationRequest resources associated with an mCODE patient and associated with cancer treatment SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
```

Comment:
> MedicationRequest resources associated with an mCODE patient and associated with cancer treatment SHOULD conform to this profile.

Why is this not a MUST?

Reply:
With the addition provision of association with cancer treatment, it can be a MUST.

Reply:
Changed.